src/PhpEnvironment/Request.php

```diff
@@ -266,8 +266,8 @@ public function setServer(ParametersInterface $server)
         $port = null;
 
         // Set the host
-        if ($this->getHeaders()->get('host')) {
-            $host = $this->getHeaders()->get('host')->getFieldValue();
+        if ($headers->get('host')) {
```

Comment:
We have then twice `->get('host')`.
What about:
```php
$headerHost = $this->getHeaders()->get('host');
if ($headerHost) {
    $host = $headerHost->getFieldValue();
    ...
```

Reply:
Ok, implemented assign first to `$headerHost`